### ISSUE TITLE
Find Bug for > 100% for Buoy

### ISSUE THREAD

**Issue by hahnd** [JCSDA/FSOI#38]
Maybe in NRL data at the end of January.  Fraction of beneficial or neutral observations.

**Comment by fcvdb** [comment]
I cannot reproduce the faulty plots. Let's close the issue for the time being. I'll test again with both the NASA and NRL data.

**Comment by hahnd** [comment]
I looked through all of the old cached cases and found 5 cases that had bad values for the Fraction of Beneficial Observations.  I deleted those from the cache and re-ran the cases.  Of the 5, only one produced bad values with the latest code.  I'll re-open this issue since we have a case.

**CASE 1**
AMSUA-N18 has -1.2% as value for Fraction of Beneficial Observations
http://ios.jcsda.org?cache_id=D762F020A2F46E138C607340E4303A079D2294F0C4C3F13BFBC4A94E6677C884
{ "start_date": "20150219", "end_date": "20150221", "centers": [ "EMC", "GMAO", "NRL" ], "norm": "dry", "platforms": "Radiosonde,Ship,Land Surface", "cycles": [ "00" ], "interval": 24, "root_dir": "/tmp/fsoi" }
